libs/shared/common/src/lib/components/switch/switch.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, EventEmitter, Input, Output, signal } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { SelectWrapper } from '../select/select.types';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'expenses-tracker-switch',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './switch.component.html'
+})
+export class SwitchComponent<T = unknown> {

```

Comment:
Rename this to SegmentedControl

Reply:
Fixed in https://github.com/usamazansari/expenses-tracker/pull/74/commits/b2d2d85a0363ef4eef0444007fe9257f3e0cb5c3